setup/system.sh

```diff
@@ -328,15 +327,14 @@ restart_service resolvconf
 
 # Configure the Fail2Ban installation to prevent dumb bruce-force attacks against dovecot, postfix, ssh, etc.
 rm -f /etc/fail2ban/jail.local # we used to use this file but don't anymore
-cat conf/fail2ban/jails.conf \
-	| sed "s/PUBLIC_IP/$PUBLIC_IP/g" \
+< conf/fail2ban/jails.conf sed "s/PUBLIC_IP/$PUBLIC_IP/g" \
```

Comment:
Given this is a multi-line set of piped commands, readability is probably better with the original

Reply:
Maybe, but it is also an error: https://github.com/koalaman/shellcheck/wiki/SC2002. Also, see [here](https://en.wikipedia.org/wiki/Cat_(Unix)#Useless_use_of_cat).